requirements/07-testing/00-intro.md

```diff
@@ -17,12 +17,12 @@ I'm going to quickly lay out the benefits of writing tests, and I'll go into det
 6. It saves time
 
 > Pro Tip:
-Writing tests for the negative paths in your code are much more valuable than writing tests for the positive paths. When time is short, start by testing the negative paths, saving the positive paths for the end. After all, bugs are rarely seen in the positive paths.
+Writing tests for the negative paths in your code is much more valuable than writing tests for the positive paths. When time is short, start by testing the negative paths, saving the positive paths for the end. After all, bugs are rarely seen in the positive paths.
 
-Before I go into detail about each benefit, I want to first debunk of a few myths about testing.
+Before I go into detail about each benefit, I want to first debunk a few myths about testing.
 
 **Myth: Testing is hard**
-Every new skill is hard at first. Just like anything else in software, once you learn how to do it, it's quick and easy – and can be quite fun & rewarding (more on that in a minute). Learn the test suite, begin thinking from a testing perspective, and it'll all fall into place and no longer be one of those activities you dread.
+Every new skill is hard at first. Just like anything else in software, once you learn how to do it, it's quick and easy – and can be quite fun & rewarding (more on that in a minute). Learn the test suite, begin thinking from a testing perspective, and it'll all fall into place and it'll be no longer be one of those activities you dread.
```

Comment:
Revert this one, as I think it read better the as was instead of the change, which reads "it'll be no longer be...". Reminds me of Shakespeare. "To be or not to be... that... is the question."

Reply:
it is not to be :P